shared/src/components/select_input.tsx

```diff
@@ -17,8 +17,10 @@ import * as React from 'react';
 import TextField from '@material-ui/core/TextField';
 import { stylesheet } from 'typestyle';
 import MenuItem from '@material-ui/core/MenuItem';
-import { INPUT_TEXT_STYLE } from '../styles';
+import { INPUT_TEXT_STYLE, ALIGN_HINT, FORM_LABEL_STYLE } from '../styles';
```

Comment:
You'll need to bump the version in the shared package.json file.

Reply:
Ah yes I always forget this